importers/djornl/manifest.schema.json

```diff
@@ -0,0 +1,52 @@
+{
+ "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Exascale parser file manifest",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "required": ["data_type", "path"],
+    "oneOf": [
+      {
+        "properties": {
+          "data_type": { "enum": ["cluster"] }
+        },
+        "required": [ "prefix" ]
+      },
+      {
+        "properties": {
+          "data_type": { "enum": [ "node", "edge" ] }
+        }
+      }
+    ],
```

Comment:
if the data type is "cluster", there must be a "prefix" field, which dictates the cluster label